### ISSUE TITLE
Failed to create custom choice validator

### ISSUE THREAD

**Issue by youyaochi** [guyskk/validr#13]
I'm trying to make a choice validator like `test_custom_validator`  in `test_custom_validator.py`,
but it will be better with "choices" parameter, so I made this:
```
def choice_validator(choices: list or tuple):
    @validator_wrap(string=False)
    def _validator(value):
        if value not in choices:
            raise Invalid('invalid choice')
        return value

    return _validator
```
and used it like:
```
SP = SchemaParser(validators={'choice': choice_validator})
SP.parse({'type?choice(["A", "B"])': 'blahblah'})
```
unfortunately this would raise exception:
```
validr._exception.SchemaError: invalid JSON value in '["A", "B"]'
```
and I find the parameters between `"()"` will be split by `","`, so there is no way to pass array variable to a custom validator?

It will be great helpful if there is an example of custom validator with parameters.

**Comment by guyskk** [comment]
I just found it's a bug in https://github.com/guyskk/validr/blob/master/validr/schema.py#L118
, you are right, thank you for the bug report!

Try this workaround:
```python
@validator(string=False)
def choice_validator(value, *choices):
    try:
        if value in choices:
            return value
    except:
        pass
    raise Invalid('invalid choice')

sp = SchemaParser(validators={'choice': choice_validator})
f = sp.parse({'type?choice("A","B","C")': 'blahblah'})
```

**Comment by youyaochi** [comment]
Thanks for the fast replay, this validator is perfect!

**Comment by guyskk** [comment]
Fixed in v0.14.1
